draft-terbu-sd-jwt-vc.md

```diff
@@ -421,37 +447,78 @@ The following is a non-normative example of a JWT Issuer Metadata including
 
 # Verifiable Presentations
 
-This specification defines the media type `vp+sd-jwt` which describes the VP-SD-JWT with the following components:
+This section defines encoding, validation and processing rules for VP-SD-JWTs.
+
+## Media Type
 
- 1. The SD-JWT from the VC-SD-JWT.
- 1. A subset of the SD-JWT Disclosures that are selectively disclosed by the Holder.
- 1. An optional holder binding JWT that proves the Holder is the intended Holder of the Verifiable Credential. Note, it is the responsibility for the Issuer to include a confirmation method in the Verifiable Credential. This process is referred to as holder binding. This enables the Holder to prove they are the intended Holder of the Verifiable Credential. Further note, it is up to the Verifier to require or to not require the Holder to prove posessions of the confirmation method.
+VP-SD-JWTs compliant with this specification MUST use the media type
```

Comment:
that is technically not feasible

Reply:
This is feasible. I doesn't say that `typ` has to be set.

Reply:
A VP-SD-JWT is a Combined Format object, not the SD-JWT contained in it.

Reply:
I removed the media type for VPs. Let's rename VP-SD-JWT in a separate PR since it would impact more sections.